### PR TITLE
[docs] Vulkan SDK usage for Android triplets

### PR DESCRIPTION
**Pull request by luncliff**
### What does your PR fix?

Add a short description/sample for `vulkan` package with NDK.

For preview: https://github.com/microsoft/vcpkg/blob/master/docs/users/android.md#using-vulkan-sdk

### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?

Will be glad to hear about the style, link usage, etc.

### CODE REVIEW

File: docs/users/android.md [hunk 1]
```diff
@@ -95,6 +95,96 @@ cd $VCPKG_ROOT
 ./vcpkg install jsoncpp:x64-android
 ````
 
+### Using Vulkan SDK
+
+VcPkg has a [`vulkan` package](https://github.com/microsoft/vcpkg/blob/master/ports/vulkan/portfile.cmake) which allows you to `find_package(Vulkan)`. To use it you have to provide `VULKAN_SDK` environment variable.
```
Comment (Cheney-W) [inline docs/users/android.md:100]:
```suggestion
Vcpkg has a [`vulkan` package](https://github.com/microsoft/vcpkg/blob/master/ports/vulkan/portfile.cmake) which allows you to `find_package(Vulkan)`. To use it you have to provide `VULKAN_SDK` environment variable.
```

**Review by Cheney-W — CHANGES_REQUESTED** [review]

**Review by Cheney-W — APPROVED** [review]

**Comment by luncliff** [comment]
Added more detailed path examples for `find_library` users. --> 4004fb7

**Comment by Cheney-W** [comment]
osgearth:x64-linux failure is a CI issue, it will be fixed by PR: https://github.com/microsoft/vcpkg/pull/16088

**Comment by vicroms** [comment]
This is very useful, thanks for the PR!

**Pull request merged**